User Engagement Lead/survey_questions.md

```diff
@@ -0,0 +1,79 @@
+# Survey questions
+
+## Readme
+
+This survey aims at evaluating users’ experiences with ESMValTool and
+discovering users’ needs.  It is meant for the users of ESMValTool in all levels
+of expertise. Some criteria to design the questions:
+
+- It does not take more than 10 (?) minutes
+- It does not have overlapping questions
+- It does not include questions with yes/no answers
+- It does not collect personal data, but it has some profiling questions
+- It covers different aspects of ESMValTool like application, usage, documentation and community
+
+The survey can be implemented in a survey app like surveymonkey that does not
+collect personal data. The owner is “user engagement team”. They will
+edit/run/evaluate the survey and communicate the results to other teams if
+needed.
+
+## Survey content
+
+Hi! We in [ESMValGroup](https://www.esmvaltool.org/), would love to hear about
```

Comment:
in ESMValGroup -> in the ESMValGroup team or development team.

Reply:
Hi @SarahAlidoost  - Very nice work. I have posted some suggested changes (accept as you wish) and some questions for clarification.